integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/indexserver/PrePrimingListener.scala

```diff
@@ -40,7 +43,15 @@ object PrePrimingEventListener extends OperationEventListener {
       false,
       true)
     if (prePrimingEvent.segment.length != 0) {
-      IndexServer.getClient.getCount(dataMapFormat)
+      try {
+        IndexServer.getClient.getCount(dataMapFormat)
+      }
+      catch {
+        case ex: Exception =>
+          ex.printStackTrace()
+          LOGGER
+            .info("Connection to Index Server Failed. Loading without prepriming in Index Server")
```

Comment:
it may not be always the connection issue, since pre priming is an additional optimization, if it fails, we just go ahead by telling the error. So you can just say, pre-priming to specific table failed and print log error like above